.github/rawWorkflows/gh-ci-alpini-parametrized-flow.txt

```diff
@@ -0,0 +1,73 @@
+  $FlowName:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        jdk: [11]
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: $TimeOut
+    outputs:
+      alpini_touched: ${{ steps.check_alpini_files_changed.outputs.alpini }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Checkout as many commits as needed for the diff
+          fetch-depth: 2
+      - name: Check if files have changed
+        uses: dorny/paths-filter@v2
+        id: check_alpini_files_changed
+        with:
+          filters: |
+            alpini:
+              - 'internal/alpini/**'
+      - uses: actions/checkout@v3
+        if: steps.check_alpini_files_changed.outputs.alpini == 'true'
+        with:
+          fetch-depth: 0
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        if: steps.check_alpini_files_changed.outputs.alpini == 'true'
+        with:
+          java-version: ${{ matrix.jdk }}
+          distribution: 'microsoft'
+          cache: 'gradle'
+      # - name: Allow Deprecated TLS versions for Alpini tests
```

Comment:
Do we need to keep this?

Reply:
Till we have the disabled legacy TLS tests (`TestSslInitializer#testLegacyClientTLS` and `TestSslInitializer#testLegacyServerTLS`), I'd prefer keeping this around. When removing those tests, we should also remove the codes used only for legacy TLS

Regarding the binary files, they are keystores and truststores for tests to test various scenarios. Ideally, we should not have them and we can generate them via gradle, but that would have required us to to understand the pecularities of each individual cert. I thought that was not needed at the moment. If you think we should remove these from the start, we can create a file like: [create-ssl-cert.sh](https://github.com/linkedin/venice/blob/main/internal/venice-common/create-ssl-cert.sh)